CometServer/Server/main.cpp

```diff
@@ -7,14 +7,14 @@
 
 int main() //TODO: Maybe factor this out into a class for better testability.
 {
-	server::AbstractServer* gameserver = &server::GameServer{};
```

Comment:
This unfortunately -but in retrospect obviously- created and destroyed a temporary `GameServer` object, which was a problem, because `ReadPackets` is currently running in an infinite loop. We should later implement an exit mechanism (but even then, we should never create temporary`GameServer` s), in fact, there's already a TODO for that.